example.docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+service:
+  environment:
+    AMAZON_S3_KEYID: AmazonS3KeyId
+    AMAZON_S3_KEY: AmazonS3Key
+    UNILOGINSECRET: UniLoginSecret
```

Comment:
@kasperg should "UNILOGINSECRET" be part of the override example?
I guess it shouldn't because it's only for talking between to your local instance?